backend/src/jobs.rs

```diff
@@ -64,6 +64,10 @@ pub fn workspaced_service() -> Router {
         .route("/getupdate/:id", get(get_job_update))
 }
 
+pub fn global_service() -> Router {
+    Router::new().route("/resume/:id", post(resume_job))
```

Comment:
I am thinking of changing a bit the spec so we have an approve/disprove GET method that would have empty payload but count as a resume payload for approve, and fail the flow for disprove